README.md

```diff
@@ -4,16 +4,21 @@ The official Elastic.io object-storage client.
 
 ## Usage
 
+This library propose you two clients: `ObjectStorage` & `ObjectStorageWrapper`. 
```

Comment:
It's nice to have a section about how to handle errors. You can check small section in our lib for reference.